pkg/apply/applier.go

```diff
@@ -176,14 +156,35 @@ func (a *Applier) Run(ctx context.Context, invInfo inventory.Info, objects objec
 				ResourceCache: resourceCache,
 			},
 		}
+		taskBuilder := &solver.TaskQueueBuilder{
+			Pruner:        a.pruner,
+			DynamicClient: a.client,
+			OpenAPIGetter: a.openAPIGetter,
+			InfoHelper:    a.infoHelper,
+			Mapper:        a.mapper,
+			InvClient:     a.invClient,
+			Collector:     vCollector,
+			ApplyFilters:  applyFilters,
```

Comment:
Not necessarily something for this PR, but could we also move more the filter setup into the solver?

Reply:
sounds like a good follow up PR. It will need a bunch of new tests.